content/python/concepts/math-module/terms/math-acos/math-acos.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'math.acos()'
+Description: 'Returns the inverse cosine (in radians) of a value between -1 and 1.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---

```

Comment:
```suggestion
---
Title: 'math.acos()'
Description: 'Returns the arc cosine (inverse cosine) of a number.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

```

Reply:
The description should be the function, and the tags should be alphabetically arranged